src/Microsoft.Extensions.DiagnosticAdapter/project.json

```diff
@@ -17,22 +17,17 @@
     "xmlDoc": true
   },
   "frameworks": {
-    "net451": {
-      "frameworkAssemblies": {
-        "System.Runtime": { "type": "build" }
-      }
-    },
-    "netstandard1.3": {
+    "netstandard1.1": {
       "dependencies": {
         "System.ComponentModel": "4.0.1-*",
         "System.Collections.Concurrent": "4.0.12-*",
         "System.Reflection.Emit": "4.0.1-*",
         "System.Reflection.Emit.Lightweight": "4.0.1-*",
         "System.Reflection.Extensions": "4.0.1-*",
-        "System.Reflection.TypeExtensions": "4.1.0-*"
```

Comment:
Yeah their choice for selecting TFMs can be very strange at times. For instance - https://github.com/aspnet/FileSystem/pull/169/files#r55793374